packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -34,60 +37,61 @@ export class Facepile extends BaseComponent<IFacepileProps, {}> {
   public static defaultProps: IFacepileProps = {
     maxDisplayablePersonas: 5,
     personas: [],
+    overflowPersonas: [],
     personaSize: PersonaSize.size32
   };
 
   private _ariaDescriptionId: string;
+  private _id: string;
 
   constructor(props: IFacepileProps) {
     super(props);
 
+    this._id = getId('Facepile');
     this._ariaDescriptionId = getId();
-
   }
+
   public render(): JSX.Element {
     let {
       overflowButtonProps,
       overflowButtonType,
     } = this.props;
     const {
+      ariaDescription,
```

Comment:
not seeing where this is used.

Reply:
Thanks, guess VSCode didn't notice that.